src/main/g8/default.properties

```diff
@@ -1,7 +1,7 @@
 name=My Akka HTTP Project
 description=This is a seed project which creates a basic build for an Akka HTTP application using Scala.
-scala_version=2.13.1
-akka_http_version=maven(com.typesafe.akka, akka-http-core_2.13, stable)
+scala_version=2.13.3
+akka_http_version=10.2.0
```

Comment:
10.2.0 will become the stable version at Maven Central.

Reply:
Yep, that should be reverted then.

Reply:
Done